From d89cdb131f04ae547f05b92fb519a5e4a90c4f01 Mon Sep 17 00:00:00 2001
From: Rover12421 <rover12421@163.com>
Date: Wed, 18 Nov 2015 17:21:45 +0800
Subject: [PATCH] static build

---
 frameworks/base/tools/aapt/Android.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/frameworks/base/tools/aapt/Android.mk b/frameworks/base/tools/aapt/Android.mk
index b701445..3276108 100644
--- a/frameworks/base/tools/aapt/Android.mk
+++ b/frameworks/base/tools/aapt/Android.mk
@@ -106,6 +106,11 @@ LOCAL_LDLIBS_linux := $(aaptHostLdLibs_linux)
 LOCAL_SRC_FILES := $(aaptMain)
 LOCAL_STATIC_LIBRARIES := libaapt $(aaptHostStaticLibs)
 LOCAL_STATIC_LIBRARIES_windows := $(aaptHostStaticLibs_windows)
+#[Rover12421]>
+ifneq ($($(my_prefix)OS),windows)
+LOCAL_CXX_STL := libc++_static
+endif
+#[Rover12421]<
 
 include $(BUILD_HOST_EXECUTABLE)
 
-- 
1.9.1

